charts/diskover/templates/deployment-server.yaml

```diff
@@ -44,8 +44,10 @@ spec:
         #   name: localtime
         # - mountPath: /etc/timezone
         #   name: timezone
+{{- if .Values.policies.enabled }}
```

Comment:
Did you check if `policies.enabled: true` will work with latest k8s version?

Reply:
on latest k8s version, enabled will be false.

Reply:
That would work. but what if we wanna have `dnsPolicy` and `restartPolicy` enabled on the new k8s?

Reply:
Removed the policies, they should be included by default